libmscore/rest.cpp

```diff
@@ -924,30 +942,57 @@ void Rest::localSpatiumChanged(qreal oldValue, qreal newValue)
       }
 
 //---------------------------------------------------------
-//   getProperty
+//   propertyDefault
 //---------------------------------------------------------
 
-QVariant Rest::getProperty(Pid propertyId) const
+QVariant Rest::propertyDefault(Pid propertyId) const
```

Comment:
Shouldn't `Pid::Substyle` be in there?

Reply:
I'm not sure, what does that do?

Reply:
The PR title says something about style. If it really adds a substyle property then it should be there.
For an example, see any staff text in Musescore. Almost at the bottom of the inspector, there's a style option, that allows you to chose with which style the text is drawn.

I haven't seen this implemented in the rest of the PR, so I guess the answer to my question

> Shouldn't `Pid::Substyle` be in there?

Is no.

Reply:
Aha, I remember now, that's the "new" (since 3.0) method for handling text styles. I don't think this number should be treated as text, though. It's just a symbol, like the numbers in time signatures.